monitoring/definitions/redis.go

```diff
@@ -19,13 +19,14 @@ func Redis() *monitoring.Container {
 				Rows: []monitoring.Row{
 					{
 						{Name: "redis-store_up",
```

Comment:
```suggestion
						{
							Name: "redis-store_up",
```

(see other observables for consistency)